### PR TITLE
Guide : Worker Thread Node JS

### PR DESCRIPTION
**Pull request by MrSlimCoder**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by viveksharmaui** [comment]
Hi guy's i have created .md for worker threads in node js :)

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@viveksharmaui Thank for adding the new article, but you also deleted 4 completely unrelated file.  Please revert the changes to the other files, so your PR only adds the new article.  Thank you.

**Comment by viveksharmaui** [comment]
> @viveksharmaui Thank for adding the new article, but you also deleted 4 completely unrelated file. Please revert the changes to the other files, so your PR only adds the new article. Thank you.

Yeah i will revert that .

File: guide/english/nodejs/worker-threads/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Worker Threads
+---
+
+## Worker Threads
+
+[Worker Threads](https://nodejs.org/api/worker_threads.html) is a Node.js default library made to enables the use of threads that execute JavaScript in parallel. They are useful for performing CPU-intensive JavaScript operations. They will not help much with I/O-intensive work. Node.js's built-in asynchronous I/O operations are more efficient than Workers can be. They became available in Node.js 10.5.0. Prior to Node.js 11.7.0.
```
Comment (xyozio) [inline guide/english/nodejs/worker-threads/index.md]: enables => enable

Comment (viveksharmaui) [reply guide/english/nodejs/worker-threads/index.md]: awesome thanks for feedback

**Review by RandellDawson — APPROVED** [review]

**Comment by viveksharmaui** [comment]
I have revert changes for deleted file , the problem was the big file name's and path in this project and git was automatically deleting those files so i add this configuration inside my git .
> git config --system core.longpaths true

**Comment by RandellDawson** [comment]
Congrats @viveksharmaui on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Comment by viveksharmaui** [comment]
> Congrats @viveksharmaui on your first pull request (PR)! 
> 
> Thank you for your contribution to the page! 
> We're happy to accept these changes, and look forward to future contributions.

i will be very happy to see my contribution in production guy's , and Thanks you so much for your kind words :) 
I'm very excited for my future contributions :100:

**Comment by viveksharmaui** [comment]
One more thing guy's i want to add guides , doc and tutorials in English and Urdu , i also wanna add materials and test incurdu for pakistani peoples but freecodecamp does not have urdu language , how can i do it ?

**Comment by RandellDawson** [comment]
We no longer are accepting any non-English guide articles.

**Comment by viveksharmaui** [comment]
No problem but whenever you want to update docs in urdu you can ask me i will also contribute in urdu .

**Pull request merged**